### ISSUE TITLE
During import/compile: Path must be a string. Received undefined...

### ISSUE THREAD

**Issue by strarsis** [sass-eyeglass/eyeglass#81]
During compilation, when importing a file, not sure where or which one yet, I get this error with latest eyeglass, downgraded node-sass (withtout the recursive import bug) and node v4.0.0:

````
path.js:8
    throw new TypeError('Path must be a string. Received ' +
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.join (path.js:477:5)
    at getPackage (/home/build/src/app/node_modules/eyeglass/lib/util/discover.js:10:20)
    at discover (/home/build/src/app/node_modules/eyeglass/lib/util/discover.js:96:22)
    at Object.getModules [as all] (/home/build/src/app/node_modules/eyeglass/lib/util/discover.js:198:19)
    at Object.ImportUtilities.getModuleByName (/home/build/src/app/node_modules/eyeglass/lib/import_utils.js:38:29)
    at Object.<anonymous> (/home/build/src/app/node_modules/eyeglass/lib/module_importer.js:66:30)
    at options.importer (/home/build/src/app/node_modules/node-sass/lib/index.js:321:31)
````

Thank you and
With best regards

**Comment by eoneill** [comment]
Stepping through the stack trace, I believe #72 will fix this issue. You can try pointing your `package.json` to my fork: [eoneill/eyeglass](https://github.com/eoneill/eyeglass)

Additionally, you might have to set the `root` directory in your Eyeglass options.

**Comment by strarsis** [comment]
Yes, the error disappeared now with the fork installed, but I get a Segmentation Fault now.
How can I further debug it?

**Comment by eoneill** [comment]
Using the forked version, you can try enabling the debug flag. Set an environment variable `DEBUG=eyeglass*` before running your compile task.

e.g.
```sh
$ DEBUG=eyeglass* broccoli build
```

You'll probably get a lot of noise, so you might need to narrow down which features are logged. See https://github.com/eoneill/eyeglass/blob/master/docs/TROUBLESHOOTING.md#logging-debug-messages

**Comment by chriseppstein** [comment]
#72 landed in `0.7.0`
